App.tsx

```diff
@@ -23,25 +24,26 @@ const Stack = createStackNavigator();
 export default function App(): JSX.Element {
     const { t } = useTranslation();
     const navigationRef = useRef<NavigationContainerRef>(null);
-    const replace = (name: string, params: any) => { //eslint-disable-line
+    const replace = (name: string, params: any) => {
```

Comment:
I think if we remove the `//eslint-disable-line` then we should fix the type for the explicit any for `params: any`

Reply:
I don't think we have to change anything, to be honest. When I run `npm run lint` locally, there are 0 errors and 0 warnings regarding the `App.tsx` file.

Reply:
Interesting. I got warnings there... okay.